src/emulator/util/src/util.cpp

```diff
@@ -110,6 +143,25 @@ std::string wide_to_utf(const std::wstring &str) {
     return myconv.to_bytes(str);
 }
 
+std::string remove_special_chars(std::string str) {
```

Comment:
are you sure this is all of them? normally you use a whitelist for sanitizing, not a blacklist